karma-ng.conf.js

```diff
@@ -157,7 +157,7 @@ function makeConfig(packageName, argv) {
     recordScreenshots: true,
   };
 
-  if (process.env.COVERAGE || process.env.CIRCLECI || process.env.CI) {
```

Comment:
do we know why this was getting used in the first place ?

Reply:
do we have to intoduce something for workflow as well

Reply:
discussed offline.